client/src/app/index.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import * as ROUTES from '../constants/routes';
+import Footer from '../components/common/Footer';
 
 function App() {
   return (
     <div className="App">
       <h1>Welcome to Art Gallery websites</h1>
       <Router>
         <Switch>
-          <Route exact path={ROUTES.HOME_PAGE} />
+          <Route exact path={ROUTES.HOME_PAGE} component={Footer} />
```

Comment:
You should create a new component which called `HomePage` for example and call `Footer` over there in addition to any other components we will call